client/lib/state-persistence/enhancer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { noop, throttle } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getState, setState } from './worker';
+
+const DEFAULT_OPTIONS = {
+	types: {
+		SERIALIZE: 'SERIALIZE',
+		DESERIALIZE: 'DESERIALIZE',
+		STATE_REPLACE: 'STATE_REPLACE'
+	},
+	throttle: 500,
+	maxAge: 604800000
```

Comment:
>could we express this in terms of some unit? appears to be 7000 days?

I guess @gwwar was on to something with [her implementation](https://github.com/Automattic/wp-calypso/blob/a8ef250c5eea2697db2c8fef667ab40e058c0686/client/state/initial-state.js#L26-L29) 😄  

The time is 7 days (`Date.now` is measured in milliseconds since epoch, not seconds)